docs/src/increase-observability/metrics/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Monitor metrics
+weight: 5
+description: See all of the live infrastructure metrics available to give you an overview of resource usage.
+aliases:
+  - /dedicated/architecture/metrics.html
+---
+
+Platform.sh projects are accompanied by live infrastructure metrics that provide an overview of resource usage for environments.  
+
+Within the console, metrics can be found for an environment under **Metrics**.
+
+The information under **Metrics** shows usage metrics for:
+
+* [Dedicated environments](../../dedicated/overview/_index.md):
+  each of the three hosts in your [N+1 configuration](../../dedicated/architecture/_index.md) and their average
+
+  ![A screenshot of what the metrics dashboard displays for Dedicated environments](/images/metrics/all-dedicated.png "0.45")
+* Grid environments: for the average of your service and app containers
+
+  ![A screenshot of what the metrics dashboard displays for Grid environments](/images/metrics/all-grid.png "0.45")
+
+## Default thresholds
+
+All of the graphs show labels for the following thresholds:
+
+* Usage that crosses _80%_ results in a **warning** label.
+* Usage that crosses _90%_ results in a **critical** label.
+* On Grid environments only, usage that crosses _100%_ results in a **burst** label.
+
+  The burst capability is available for containerized environments
+  and allows a container to get more resources than it's allocated.
+  Burst is considered useful for infrequent activities that cause usage spikes,
+  but these additional resources aren't guaranteed.
+
+### Recommendations
+
+The default thresholds aim to give you an idea of when your hosts/containers are close to running out of resources.
+The impact differs based on your specific apps and service.
+The values of the thresholds is purely informational.
+
+#### Dedicated environments
+
+For Dedicated environments, the thresholds are set for each host.
+If the resources are high and hovering close to the 100% threshold,
+you might want to consider:
+
+* [Optimize your code](../integrate-observability/_index.md) (if possible)
+* [Increase your plan](../../overview/pricing/_index.md)
+
+#### Grid environments
+  
+For Grid environments, the thresholds are set for each container.
+If the resources are high and hovering close to the 100% threshold,
+you might want to consider:
+
+* [Optimize your code](../integrate-observability/_index.md) (if possible)
+* Change your [app size](../../configuration/app/app-reference.md#sizes)
+  or [service size](../../configuration/services/_index.md#size)
+* [Increase your plan](../../overview/pricing/_index.md)
```

Comment:
I wouldn't link to the console generally as part of the call to action here. The pricing page needs improvement, but that's a separate issue. As above, I think the details of how and why to do things should be included there rather than here.